doc/questions.md

```diff
@@ -0,0 +1,55 @@
+# ros-get questions
+
+- *1. where do these packages come from??*
+  - Would be the ros-distro story, but needs a good place in the manual.
+  - perhaps an example and a eference to the REP.
```

Comment:
You create a rosdistro according to [REP 143](http://www.ros.org/reps/rep-0143.html). Example for the tue-robotics software: https://github.com/rayman/rosdistro

I made one modification, the `packages` field [example](https://github.com/Rayman/rosdistro/blob/25102be51beeaa4920cb0cdf542c002f37ae5c74/kinetic/distribution.yaml#L16) is added for the source packages field. When this field is present, installation with ros-get is possible.